locales/en.json

```diff
@@ -255,5 +255,9 @@
   "Total Deducted (437)": "Total Deducted (437)",
   "Total Credits (482)": "Total Credits (482)",
   "Total Refund (484)": "Total Refund (484)",
-  "I confirm that, as far as I know, all the information in this tax return is accurate": "I confirm that, as far as I know, all the information in this tax return is accurate"
+  "I confirm that, as far as I know, all the information in this tax return is accurate": "I confirm that, as far as I know, all the information in this tax return is accurate",
+  "Ontario trillium benefits": "Ontario trillium benefits",
+  "Detailed breakdown of tax refund": "Detailed breakdown of tax refund",
+  "Medical Expenses (330)": "Medical Expenses (330)",
+  "Net Ontario tax (428)": "Net Ontario tax (428)"
 }
```

Comment:
Can you remove this? Let's avoid changing these files while both Cayce and Elise are working on them.